openwisp_controller/config/apps.py

```diff
@@ -83,18 +84,34 @@ def connect_signals(self):
             dispatch_uid='cert_update_invalidate_checksum_cache',
         )
 
-    def add_default_menu_items(self):
-        menu_setting = 'OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'
-        items = [
-            {'model': get_model_name('config', 'Device')},
-            {'model': get_model_name('config', 'Template')},
-            {'model': get_model_name('config', 'Vpn')},
-        ]
-        if not hasattr(settings, menu_setting):
-            setattr(settings, menu_setting, items)
-        else:
-            current_menu = getattr(settings, menu_setting)
-            current_menu += items
+    def register_menu_groups(self):
+        register_menu_group(
+            position=11,
+            config={
+                'label': 'Configurations',
```

Comment:
I think we should use controller here, that's more clear, what do you think @nemesisdesign @niteshsinha17?

Reply:
@atb00ker configurations is more straightforward, but since the device list is most probably the most accessed page it may be a good idea to create a direct link to it as first element after the home/dashboard button.

@niteshsinha17 BTW we just added device groups in master, can you please try the following:

- Devices
- Configuration
  - Templates
  - VPN Servers
  - Device Groups

Reply:
Sure, I will update this.